SwordImportExportPlugin.inc.php

```diff
@@ -202,7 +242,7 @@ public function display($args, $request) {
 					foreach ($articleIds as $articleId) {
 						$publishedArticle = $publishedArticleDao->getByArticleId($articleId);
 						try {
-							$deposit = new OJSSwordDeposit($publishedArticle);
+ 							$deposit = new OJSSwordDeposit($publishedArticle);
```

Comment:
I think this change is inadvertent (it introduces a leading space). But if we're going to support PPS, some of the language here needs to be generalized -- e.g. articleId should be submissionId, and `OJSSwordDeposit` should either be generalized or only be referenced within a conditional check to ensure that we're using OJS.